### PR TITLE
Default store, recenter location button

### PR DESCRIPTION
**Pull request by tommypoa**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR
- Dollar Plus Howard will be set as the default store if
   - User is more than 100 miles away
   - Location permission is disabled
- Design changes for the above default case implemented
- Recenter location button added
   - Implemented in #11 
- Store list re-orders if you change your location, then hit the "center location" button

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

## Relevant Links

[//]: # "Optional - copy links to any tutorial or documentation that was useful to you when working on this PR"

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"
#11 
## How to review

[//]: # "The order in which to review files and what to expect when testing locally"
- Manually set location to be far away and see if the default case is triggered/look for edge cases
- Visual alignment to Figma
- Test recenter button

## Next steps

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"
- Ordering of storelist during default option (distance from Dollar Plus Howard/default store)
- Position of recenter button

## Tests Performed, Edge Cases

[//]: # "Hopefully we will add a testing suite/CI soon, but until then note down the steps you took to test locally"

### Screenshots

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"
- Recenter button right below hamburger for now. Icon of recenter taken from FontAwesome.
image

CC: @wangannie

[//]: # "This tags in both Annies as a default. Feel free to change, or add on anyone who you should be in on the conversation."

### CODE REVIEW

File: screens/map/MapScreen.js [hunk 1]
```diff
@@ -204,6 +225,7 @@ export default class MapScreen extends React.Component {
             </Subhead>
           </SearchBar>
         </NavHeaderContainer>
+        <CenterLocation callBack={() => this._findCurrentLocationAsync()} />
```
Comment (wangannie) [inline screens/map/MapScreen.js:228]: Definitely nonessential but are there easy ways to make this transition smoother?

File: screens/auth/SignUpScreen.js [hunk 2]
```diff
@@ -9,19 +9,19 @@ import AuthTextField from '../../components/AuthTextField';
 import {
   BigTitle,
   ButtonLabel,
-  FilledButtonContainer
+  FilledButtonContainer,
```
Comment (anniero98) [inline screens/auth/SignUpScreen.js:12]: Not sure why these changes didn't happen in #48 but it's nbd I think

File: screens/map/MapScreen.js [hunk 3]
```diff
@@ -79,7 +86,13 @@ export default class MapScreen extends React.Component {
         longitude: location.coords.longitude,
         ...deltas,
       };
-      this.setState({ locationErrorMsg: null, location, region });
+      // Don't re-animate if we're using the default store
```
Comment (anniero98) [inline screens/map/MapScreen.js:89]: I tried to make this work with the default store, but the resetting of the location in `orderStoresByDistance` for only the default store produced some really odd behavior, so I think it's ok to lose the animation for the default store.
